packages/driver/src/cy/jquery.ts

```diff
@@ -9,6 +9,14 @@ const remoteJQueryisNotSameAsGlobal = (remoteJQuery) => {
 
 // eslint-disable-next-line @cypress/dev/arrow-body-multiline-braces
 export const create = (state) => ({
+  $$ (selector, context) {
+    if (context == null) {
+      context = state('document')
+    }
+
+    return $dom.query(selector, context)
+  },
+
```

Comment:
FYI - this was pulled from src/cypress/cy.js https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/cy.ts#L349

This just is missing from our documented utils: https://docs.cypress.io/api/utilities/$